src/components/ImageUpload/ImageUploadButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRef } from 'react';
+
+import Icon from '../Icon';
+
+interface Props {
+  setImageUrl: (imageUrl: string) => void;
+  onChange?: (file: File) => void;
+}
+
+const ImageUploadButton = ({ setImageUrl, onChange }: Props) => {
+  const inputElement = useRef<HTMLInputElement>(null);
+
+  const handleImageSelect = () => {
+    if (!inputElement.current) {
+      return;
+    }
+    if (!inputElement.current.files) {
+      return;
+    }
+    const [file] = inputElement.current.files;
+    if (!file) {
+      return;
+    }
+    if (onChange) {
+      onChange(file);
+    }
```

Comment:
아래처럼 동일하게 동작하는 코드를 좀 더 짧게 작성할 수 있을 것 같아요 ㅎㅎ
취향에 따라 충분히 달라질 수 있는 부분이라 반영하지 않으셔도 좋습니다~!
```suggestion
    onChange && onChange(file);
```